### DISCUSSION TITLE
Space key shortcut representation

### DISCUSSION THREAD

**Discussion by noritada** [zed-industries/zed#7316] [Feature Requests]
When I saw the shortcut key for `project panel: open` for the first time, I thought it was an underscore (`_`) or hyphen (`-`) when it should have been recognized as a space. Also, in the menu, the space key seems to be labeled as "Space". So I think it would be better for "Space" to be used also in the command palette.

![ss 2024-02-02 1 53 18](https://github.com/zed-industries/zed/assets/20850/8c3e4940-0114-410d-bd43-444be0dac488)
![ss 2024-02-02 1 53 43](https://github.com/zed-industries/zed/assets/20850/228ca182-69d4-4826-8233-41435ec9c7f5)

I would like to submit a PR for the change if it is necessary, but I think I should make confirmation before that if the change is necessary in the first place.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
